Makefile

```diff
@@ -2,9 +2,19 @@ ci: clean tools deps lint
 
 deps:
 	r10k puppetfile install --verbose --moduledir modules
+	mkdir -p inspec/profiles/inspec-aem
+	mkdir -p inspec/profiles/inspec-aem-aws
+	mkdir stage
+	cd stage
+	wget https://github.com/shinesolutions/inspec-aem/archive/master.tar.gz
+	tar -xzf master.tar.gz ../inspec/profiles/inspec-aem/
+	wget https://github.com/shinesolutions/inspec-aem-aws/archive/master.tar.gz
+	tar -xzf master.tar.gz ../inspec/profiles/inspec-aem-aws/
 
 clean:
 	rm -rf pkg
+	rm -rf inspec/
```

Comment:
Move inspec/ to test/inspec/ , in order to get inspec profiles bundled, modify `.pmtignore` file so that it ignores the files under test directory, except test/inspec/ directory.